libs/pricing_model/Makefile

```diff
@@ -0,0 +1,13 @@
+all: 1.svg 2.svg 3.svg 4.svg 1.png 2.png 3.png 4.png
```

Comment:
Aah yes, broke it when renaming.

Would probably be easier to define the binary as phony target and always invoke it through `cargo run` instead of querying where is the target directory.

Reply:
Using `cargo metadata` would be the best way, but the `../../target/debug/pricing_model` works as well. We have other dependencies towards `target` being in the checkout so not inclined to make this anymore complicated.

Reply:
Fixed in https://github.com/neondatabase/neon/pull/2714/commits/ae5d84010b2fe99c7c0c17f8adf62a6305c25df4 if you'd like to retry?

Reply:
Works like a charm now.

Reply:
One thing we could consider doing is to add some gitignore for the new files:
```
work/neon/neon (6f6bfb3) * ❯ git s
## HEAD (no branch)
?? libs/pricing_model/1.png
?? libs/pricing_model/1.svg
?? libs/pricing_model/2.png
?? libs/pricing_model/2.svg
?? libs/pricing_model/3.png
?? libs/pricing_model/3.svg
?? libs/pricing_model/4.png
?? libs/pricing_model/4.svg
```

maybe, put them into a subdirectory and ignore that?

Reply:
Would you be ok with local git ignore for `*.{png,svg,dot}`? Have to admit I don't know an elegant makefile solution to move them all in a subdirectory.

Reply:
I wonder, if we really need a makefile instead of a script then, but up to you.
If we don't plan to generate those images frequently, might be ok to leave everything as is already, I imagine.